db-commons/src/main/scala/org/bitcoins/db/CRUDAction.scala

```diff
@@ -18,4 +18,43 @@ abstract class CRUDAction[T, PrimaryKeyType](implicit
     createAllAction(Vector(t))
       .map(_.head)
   }
+
+  def updateAction(t: T): DBIOAction[T, NoStream, Effect.Write] = {
+    updateAllAction(Vector(t)).map(_.head)
```

Comment:
Should match on the headOption like we did before so we can throw a more descriptive error